src/EnergyPlus/StringUtilities.hh

```diff
@@ -137,6 +137,8 @@ inline auto nth_occurrence(std::string_view input_str, std::string_view search_s
     return pos;
 }
 
+// Returns the position **after** the nth occurrence of the search_char, eg:
+// eg: Given a string test="aa,bb,cc", nth_occurrence(test, ',', x), for x = 1 it gives 3, and for x = 2 it gives 6
```

Comment:
Docstring for nth_occurrence, as it is perhaps confusing.

Reply:
Helpful, thanks.